README.md

```diff
@@ -28,21 +28,22 @@ New here or looking to install? Check [our website](https://randovania.github.io
 
 <!-- Begin WELCOME -->
 
-Here you will be able to randomize many aspects of either game, while still being ensured it's possible to
-finish without any trick or glitch! What can be randomized?
+Randovania can randomize many aspects of its supported games, all while still ensuring they're completable 
+without any unintended tricks or glitches. Its features include:
 
-* Randomize what can be found in each pickup location, including major upgrades, expansions, keys and artifacts.
+* Randomizing what can be found in each pickup location. Weapons, keys, and more can end up in
+  completely new places.
 
-* Play with multiple people, via multiworld sessions. Your pickups will be shuffled among the games of
-everyone involved, no matter what game they're playing!
+* Use Multiworld sessions to shuffle items between multiple separate games, alone or with friends.
+  All Multiworld games are compatible with each other - mix and match as you like!
 
-* Randomize where each teleporter goes, or what you need to unlock a translator gate. In either case,
-there's advanced options for how they're shuffled.
+* Randomize how areas connect to each other, or what resources are required to travel between areas.
```

Comment:
Having another `each other` right after the MWs `each other` reads a bit weird.

Reply:
changing the second one to `one another`

Reply:
I might say "transporter" or "region transport", or something else. In both echoes, dread, SR teleporters all mean different things lol.

Reply:
I'm confused - the new version doesn't use the word "teleporter" at all. I deliberately want to avoid naming what kind of thing links areas (transporter, teleporter, door, boat, whatever) to keep it more game-neutral

Reply:
> I'm just stupid and was reading the red text :DSLOL: